benefits/core/templates/core/base.html

```diff
@@ -87,8 +87,8 @@
             {% endblock inner-content %}
           </div>
           {% block call-to-action %}
-            <div class="row d-flex justify-content-lg-end pt-8">
-              <div class="col-lg-3 offset-2 offset-sm-2 offset-lg-0 col-sm-8 col-8">
+            <div class="row d-flex justify-content-center pt-8">
+              <div class="col-12 col-lg-6">
                 {% block call-to-action-button %}
                 {% endblock call-to-action-button %}
```

Comment:
This block is only used by `enrollment-index`.

Reply:
Note for #2540

Reply:
Yeah and this is turning into a true PR rabbit hole, but in the _next_ PR for Enrollment Success/Logged Out, https://github.com/cal-itp/benefits/pull/2579/files#r1876743474, I moved some code out of `call-to-action` to clean it up further. After https://github.com/cal-itp/benefits/pull/2579/files#r1876743474, in #2540 will be able to consolidate some of the blocks in base.